marketplace-core/src/main/java/org/pentaho/marketplace/endpoints/MarketplaceService.java

```diff
@@ -75,7 +76,7 @@ public StringOperationResultDTO hello() {
   @GET
   @Path( "/plugins" )
   @Produces( MediaType.APPLICATION_JSON )
-  public IterablePluginOperationResultDTO getPlugins() {
+  public Response getPlugins() {
```

Comment:
Do not change the method return type. If you need to change the HTTP properties of the Response add "@Context HttpServletResponse response" to the method arguments. See DET for an example: https://github.com/webdetails/det/blob/master/pentaho-det-core/src/main/java/org/pentaho/det/impl/endpoints/DataSourceService.java#L72-L83